changelog.md

```diff
@@ -1,3 +1,6 @@
+# 0.0.6
+ * Add HDFSArchiver, a hdfs tool.
```

Comment:
Could you please add a line about fixing the hdfs-rsync tests and correcting a bug in update mode? Many thanks :)

Reply:
This has been added.